src/web/core/src/main/java/org/geoserver/web/wicket/modal/modal.css

```diff
@@ -14,251 +14,120 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-div.wicket-mask-dark {  
-    position: fixed;    
-    top: 0px; 
-    left: 0px; 
-    width: 100%; 
-    height: 100%;
-    opacity: 0.1; 
-    background-color: black; 
-    filter: alpha(opacity=10); 
-    background-image: url('transparent2.png');
-}
-
-div.wicket-mask-transparent {
-    position: fixed;    
-    top: 0px; 
-    left: 0px; 
-    width: 100%; 
-    height: 100%;
 
-    background-image: url('transparent1.gif');
-}
-
-div.wicket-modal {  
-    position: fixed;
-    _position: absolute;
-    z-index: 20001; 
-    visibility: hidden;
-}
 
 
-div.wicket-modal div.w_top_1 {  
-    width: 100%;    
-    overflow: hidden;
-}
 
-div.wicket-modal div.w_top {
-    height: 11px; 
-    background-position: 0px -16px; 
-    overflow: hidden;
-    cursor:n-resize;
-    padding: 0px; margin: 0px;      
+div.wicket-modal {
+    position: absolute;
+    z-index: 20001;
+    visibility: visible;
+    background-color: white;
```

Comment:
cannot remember if there is a theme variable for background-color that should be used here.

Checking and the answer is *no*, it is just a good idea that is not used on geoserver.css.